contracts/token/TokenTimelock.sol

```diff
@@ -5,21 +5,21 @@ import './ERC20Basic.sol';
 
 /**
  * @title TokenTimelock
- * @dev TokenTimelock is a token holder contract that will allow a 
+ * @dev TokenTimelock is a token holder contract that will allow a
  * beneficiary to extract the tokens after a given release time
  */
 contract TokenTimelock {
-  
+
   // ERC20 basic token contract being held
   ERC20Basic token;
 
   // beneficiary of tokens after they are released
   address beneficiary;
 
   // timestamp when token release is enabled
-  uint releaseTime;
+  uint64 releaseTime;
 
-  function TokenTimelock(ERC20Basic _token, address _beneficiary, uint _releaseTime) {
+  function TokenTimelock(ERC20Basic _token, address _beneficiary, uint64 _releaseTime) {
```

Comment:
@maraoz I noticed that in some of the implementations, time is expressed as a uint64 as opposed to a uint256 (LimitedTransferToken, VestedToken). I sought to follow that standard. Is there any reason for using uint64 as opposed to 256?